field/interface.go

```diff
@@ -0,0 +1,18 @@
+package field
+
+import "fmt"
+
+// An Element of a prime-order field.
+type Element[Operand any] interface {
```

Comment:
I think we need a few more methods to allow reading from `[]byte` and possibly also writing back to `[]byte`.

Reply:
Also, I think `CmpUint32` or similar would be good since I do a lot of comparisons against `0` and `1`, and possibly other values.

Reply:
If comparing with 0 and 1 are done frequently how's dedicated `IsZero()` and `IsOne()` methods?